makefile

```diff
@@ -207,8 +207,13 @@ $(buildDir)/make-tarball:cmd/make-tarball/make-tarball.go
 	@mkdir -p $(buildDir)
 	@GOOS="" GOARCH="" $(gobin) build -o $@ $<
 	@echo $(gobin) build -o $@ $<
+
+
+dist-staging: export STAGING_ONLY := 1
```

Comment:
Would it make more sense to use a `make` variable here instead of an environment variable. Make could check that variable instead of the environment variable.